homeassistant/components/camera/onvif.py

```diff
@@ -21,11 +24,17 @@
     async_aiohttp_proxy_stream)
 from homeassistant.helpers.service import extract_entity_ids
 
+
+def zeep_pythonvalue(self, xmlvalue):
+    return xmlvalue
+
+
+zeep.xsd.simple.AnySimpleType.pythonvalue = zeep_pythonvalue
```

Comment:
Where is this used?

3rd party non core requirements must be imported in the functions that use them.

Reply:
I believe it was required for the lib to be used, but will try to use without.